src/Parameters/HooksCreateParameters.php

```diff
@@ -18,102 +18,89 @@
  */
 namespace BigBlueButton\Parameters;
 
+/**
+ * @method string getCallbackURL()
+ * @method HooksCreateParameters setCallbackURL(string $url)
+ * @method string getMeetingID()
+ * @method HooksCreateParameters setMeetingID(string $id)
+ * @method bool|null isGetRaw()
+ * @method HooksCreateParameters setGetRaw(bool $getRaw)
```

Comment:
```
@method HooksCreateParameters setGetRaw(bool $getRaw)
```
We could also write this as

```
@method $this setGetRaw(bool $getRaw)
```

Would probably easier to read? What do you think?

Reply:
I would prefer $this as the makes it clear, that the current instance is returned

Reply:
I will change it asap.